Tree.h

```diff
@@ -20,7 +20,7 @@ template <class T>
 class PartitionNode : public TreeNode<T> {
  public:
   PartitionNode(int fid, T fv)
-    : fid_(fid), fv_(fv),
+    : fid_(fid), fv_(fv), fvote_(0.0),
```

Comment:
fv_ is the split value.. fvote_ is the "leaf value" if the intermediate node does not split further.